src/_variables.less

```diff
@@ -124,6 +124,12 @@
 @toggleSwitchColor: #fff;
 @toggleSwitchOffPosition: translateX(calc(@toggleWidth * -0.07));
 @toggleSwitchOnPosition: translateX(calc(@toggleWidth * 0.4));
+
+@toggleSwitchOffPositionSmall: translateX(calc(@toggleWidthSmall * -0.18));
+@toggleSwitchOnPositionSmall: translateX(calc(@toggleWidthSmall * 0.29));
```

Comment:
Change this to `0.39`

Reply:
These new values are breaking the .xsmall-btn toggle button's layout.  
![buttons on](https://user-images.githubusercontent.com/68125296/118548387-dcba5400-b71f-11eb-83dc-05f357e8d7b2.png)
![buttons off](https://user-images.githubusercontent.com/68125296/118548404-e17f0800-b71f-11eb-9d12-d19930226f91.png)

Reply:
please make the change anyway and let's see it